CHANGELOG.md

```diff
@@ -6,11 +6,17 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+======
+
 ### New features
 
 - If you know the content type of a file to upload via `overwriteFile` or `saveFileInContainer`,
   you can now manually set it using the `contentType` property in their `options` parameters.
 
+- 'getContainedResourceUrlAll': a function that returns the urls for each resource contained in
+  the given container
+  =
+
```

Comment:
Likewise:

```suggestion
- 'getContainedResourceUrlAll': a function that returns the urls for each resource contained in
  the given container

```

(Also removed the whitespace in between the bulleted list, IIRC it won't display them as a single list otherwise.)